airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -36,6 +36,12 @@
         "description": "Password associated with the username.",
         "type": "string",
         "airbyte_secret": true
+      },
+      "ssl": {
+        "title": "SSL Connection",
+        "description": "Encrypt data using SSL.",
```

Comment:
how would the user know if they have SSL enabled? is it running by default in postgres?

Reply:
No, you need to enable SSL.

Reply:
> how would the user know if they have SSL enabled?

I think users should know if their databases have SSL enabled or not. Maybe adding a new topic in Security's FAQ to help users understand the implication of SSL.

For example, 5tran only support SSL/TLS connections to databases, a page about security [link](https://fivetran.com/docs/security):
> Connections to customers' database sources and destinations are SSL encrypted by default.
> IMPORTANT: You must have TLS enabled on your database to connect to Fivetran.

and Stitch [link](https://www.stitchdata.com/docs/account-security/data-encryption#ssl-connections):
> Connections to these integrations and destinations will attempt to use verified SSL with no action required on your part.
> For some integrations - for example, a database hosted on your server - Stitch may support configurable SSL. To use SSL with a database Stitch supports, the database must be configured to support and allow SSL connections.